scripts/audit_settlement.py

```
@@ -0,0 +1,174 @@
+            signature_type=sig_type,
+            funder=funder,
+        )
+        creds = client.create_or_derive_api_creds()
+        client.set_api_creds(creds)
+        res = client.get_balance_allowance(BalanceAllowanceParams(asset_type=AssetType.COLLATERAL, signature_type=sig_type))
+        return res
+    except Exception as e:
+        return {"error": str(e)}
+
+
+def check_relayer_status(tx_id: str = None) -> dict:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)